cmd/shortener/main.go

```diff
@@ -51,7 +51,7 @@ func main() {
 
 	err := config.InitConfig()
 	if err != nil {
-		os.Exit(1)
+		logger.Log.Fatal("Ошибка чтения файла конфигурации")
```

Comment:
Тут потеряна ошибка:

```suggestion
		logger.Log.Fatalf("Ошибка чтения файла конфигурации: %v", err)
```

Reply:
Добавил две разные ошибки, добавил их в логгер